builtin/credential/okta/backend.go

```diff
@@ -84,6 +84,11 @@ func (b *backend) Login(req *logical.Request, username string, password string)
 	var allGroups []string
 	// Import the custom added groups from okta backend
 	user, err := b.User(req.Storage, username)
+	if err != nil {
+		if b.Logger().IsDebug() {
```

Comment:
Sort of unrelated, but why do we need to check the logger's level? Does it not respect the level set with `SetLevel()` so that `logger.Debug()` doesn't print if the logger is set at a higher level?